neuralmonkey/encoders/image_encoder.py

```diff
@@ -24,7 +24,8 @@ def __init__(self, dimension, output_shape, data_id):
         self.encoded = tf.tanh(tf.matmul(self.flat, project_w) + project_b)
 
         self.attention_tensor = None
-        self.attention_object = None
+        self.attention_object_train = None
+        self.attention_object_runtime = None
 
```

Comment:
I don't know whether we've implemented attention in this encoder. It was not used in our submission for the multimodal task, so it might be safe to ignore this for now.